### DISCUSSION TITLE
Display a Breakdown of Token Usage in the Current Context

### DISCUSSION THREAD

**Discussion by k-inoway** [open-webui/open-webui#8136] [Ideas]
![image](https://github.com/user-attachments/assets/22540ff5-438a-4649-94c2-b223ba42fc6a)

In lobe-chat, there is a feature that allows users to see the breakdown of token usage for the conversation context and system prompts.
I think having a similar feature in open-webui would be helpful to prevent excessive and unintended token usage.

Could this feature be implemented in open-webui as well? It would be greatly beneficial for better token management.

**Comment by guenhter** [comment]
Isn't this enough:

![image](https://github.com/user-attachments/assets/a54bed33-b824-47b3-ace6-db2b0c2945f6)

**Comment by k-inoway** [comment]
thx. Got it, looks like this feature is already available. I'll go ahead and close this discussion.
